src/py/mat3ra/made/tools/utils.py

```diff
@@ -95,3 +95,73 @@ def get_norm(vector: List[float]) -> float:
         float: The norm of the vector.
     """
     return float(np.linalg.norm(vector))
+
+
+# Condition functions:
+# TODO: Predefined equations can be exported using a factory or enum
+def is_point_in_circle(x=0, y=0, r=1) -> Callable[[List[float]], bool]:
```

Comment:
def _is_2d_point_in_circle(x, y)
def _is_2d_point_in_rectangle(....)

def is_coordinate_in_a_cylinder(coordinate,  z_min = 0, z_max = 1) in crystal coords for z
def is_coordinate_in_a_box(coordinate, z_min = 0, z_max = 1)

Reply:
all functions exported from here should have a 3d coordinate as argument and return Boolean